### PR TITLE
Public Schedule (#108)

### PR DESCRIPTION
**Pull request by stefancherubin**
Closes: https://github.com/hacknyu/website/issues/108

### CODE REVIEW

File: src/js/modules/core/components/Schedule.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    fontSize: "1.4rem",
+    textAlign: "center"
+  },
+  statement: {
+    lineHeight: "1.2em"
+  },
+  header: {
+    maxWidth: "750px"
+  },
+  scheduleSection: {
```
Comment (NicholasLYang) [inline src/js/modules/core/components/Schedule.tsx]: any reason this is prefixed?

File: src/js/modules/core/components/HomePage.tsx [hunk 2]
```diff
@@ -77,6 +82,9 @@ const HomePage: React.FunctionComponent<Props> = ({ classes }) => {
           <img className={classes.curvedTop} src="/img/semicircle.svg" />
           <FAQ />
         </div>
+        <div className={classes.scheduleSection}>
+        <Schedule />
```
Comment (NicholasLYang) [inline src/js/modules/core/components/HomePage.tsx]: Small thing, but let's indent this

**Comment by NicholasLYang** [comment]
Could we use the space a little better? The page is getting really large vertically, so I think a 3 column option on larger screens could be good. Basically, let's set the `flexDirection` to row by default then set a breakpoint that makes it column.

File: src/js/modules/core/components/Schedule.tsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    flexDirection: "column",
+    alignItems: "center",
+    backgroundColor: theme.secondBackground,
+    fontSize: "1.4rem",
+    textAlign: "center"
+  },
+  statement: {
+    lineHeight: "1.2em"
+  },
+  scheduleSection: {
```
Comment (NicholasLYang) [inline src/js/modules/core/components/Schedule.tsx]: Can we remove the `schedule-` prefix?

Comment (stefancherubin) [reply src/js/modules/core/components/Schedule.tsx]: Sure, I can rename `scheduleSection` to `eventTimeline,` and `scheduleList` to `dailyEvents.` Would that work?

Comment (NicholasLYang) [reply src/js/modules/core/components/Schedule.tsx]: sure

**Pull request merged**